python/src/opendp/polars.py

```diff
@@ -43,39 +74,73 @@ def noise(
     def laplace(self, scale: float | None = None):
         """Add Laplace noise to the expression.
 
-        If scale is None it is filled by `global_scale` in :py:func:`opendp.measurements.make_private_lazyframe`.
+        If scale is None it is filled by ``global_scale`` in :py:func:`opendp.measurements.make_private_lazyframe`.
 
-        :param scale: Noise scale parameter for the Laplace distribution. `scale` == standard_deviation / sqrt(2).
+        :param scale: Noise scale parameter for the Laplace distribution. ``scale == standard_deviation / sqrt(2)``
+
+        :example:
+
+        >>> import polars as pl
+        >>> expression = pl.col('numbers').dp.laplace()
```

Comment:
While this makes an expression, the expression would not be accepted by the lib. I'd go with `pl.len().dp.laplace()` instead. Similarly below for `gaussian` and above for `noise` (which doesn't have an example yet).